wifiphisher/deauth.py

```diff
@@ -7,8 +7,8 @@
 
 import threading
 import scapy.layers.dot11 as dot11
-from scapy.all import *
-from constants import *
+import scapy.arch.linux as linux
+import wifiphisher.constants as constants
```

Comment:
Relative import 'constants', should be 'wifiphisher.constants' (relative-import)